src/extension.ts

```diff
@@ -179,17 +180,33 @@ export function activate(context: vscode.ExtensionContext) {
         return;
       }
 
-      const uri = vscode.Uri.parse(`couchbase:/${node.bucketName}/${node.scopeName}/${node.collectionName}/${documentName}.json`);
-      memFs.writeFile(
-        uri,
-        Buffer.from("{}"),
-        { create: true, overwrite: true }
-      );
+        const uri = vscode.Uri.parse(`couchbase:/${node.bucketName}/${node.scopeName}/${node.collectionName}/${documentName}.json`);
+        try {
+          const result = await node.connection.cluster
+            ?.bucket(node.bucketName)
+            .scope(node.scopeName)
+            .collection(node.collectionName)
+            .get(documentName);
+          memFs.writeFile(
```

Comment:
We're using `memFs.writeFile` in both the try and catch blocks with the only difference being the document body. Could we move this to outside of the try/catch to only call it once, with either the existing document body or an empty body?